resources/benchmark-runner.mjs

```diff
@@ -23,15 +23,21 @@ function getParent(lookupStartNode, path) {
 class Page {
     constructor(frame) {
         this._frame = frame;
+        this._leakedLayout = {};
     }
 
     getLocalStorage() {
         return this._frame.contentWindow.localStorage;
     }
 
     layout() {
-        const body = this._frame.contentDocument.body.getBoundingClientRect();
-        this.layout.e = document.elementFromPoint((body.width / 2) | 0, (body.height / 2) | 0);
+        const body = this._frame ? this._frame.contentDocument.body : document.body;
+        const bodyRect = body.getBoundingClientRect();
+        if (params.layoutMode === "getBoundingClientRect")
+            this._leakedLayout.sum = bodyRect.width;
```

Comment:
I don't know if that makes a difference, but the previous code was using `.height`.

Reply:
ups, yeah let's stick to the old code.